apps/starter/src/ensemble/scripts/hello.js

```diff
@@ -1,4 +1,7 @@
+// eslint-disable-next-line no-console
 console.log("hello data manipulation! check out the response below:");
+// eslint-disable-next-line no-console, no-undef
 console.log(response);
 
+// eslint-disable-next-line no-undef
```

Comment:
Make sure your IDE is picking up the eslint config for this project, it's set to ignore this folder